core/src/main/java/org/apache/accumulo/core/fate/ZooStore.java

```diff
@@ -523,7 +526,7 @@ public long timeCreated(long tid) {
     verifyReserved(tid);
 
     try {
-      Stat stat = zk.getZooKeeper().exists(getTXPath(tid), false);
+      Stat stat = zk.exists(getTXPath(tid), false);
```

Comment:
Is there a reason that this was not changed to use `zrw`? If it's changed, then I think we can remove the `zk` instance variable.